scripts/ci-install.sh

```diff
@@ -23,10 +23,13 @@ export PATH="$HOME/miniconda/bin:$PATH"
 hash -r
 conda config --set always_yes yes --set changeps1 no
 conda update -q conda
-conda create -y -n test python=3.7
+conda install pip
```

Comment:
Is this necessary?

Reply:
Apparently, yes, because it fails if it's not there.